cmd.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+import Range from 'range-sdk';
+import { join, resolve } from 'path';
+import { writeFileSync, statSync } from 'fs';
+
+import pkg from 'json-2-csv';
+const { json2csvAsync } = pkg;
+
+import { createRequire } from 'module';
+const meow = createRequire(import.meta.url)('meow');
+
+import { log, fatal } from './lib/printer.js';
+import exportCheckins from './lib/export-checkins.js';
+
+const commands = {
+  'check-ins': exportCheckins,
+};
+
+const cli = meow(
+  `
+  Usage
+    $ range-export <options> <cmd>
+
+  Commands
+    check-ins    Export check-in data
+
+  Options
+    --after, -a   Return data after this ISO timestamp
+    --before, -b  Return data before this ISO timestamp
```

Comment:
What's your vision for branching this out as we add more commands? Eg, not all options will map to all commands.

Reply:
Hadn't thought that far ahead. I've seen Options be a union of all args. I've also seen `range-export check-ins --help`, but not sure `meow` works with that.